wallet/bdkgo/bdkgo.go

```diff
@@ -0,0 +1,1548 @@
+package bdkgo
```

Comment:
Since cargo is now a dependency, could we add build tags to the relevant files?

```diff
[I] calvin@bitcoin ~/b/g/u/g/s/g/u/utreexod ((4236f17f))> git diff
diff --git a/wallet/bdkgo/bdkgo.go b/wallet/bdkgo/bdkgo.go
index 8d48f376..6bf8aed4 100644
--- a/wallet/bdkgo/bdkgo.go
+++ b/wallet/bdkgo/bdkgo.go
@@ -1,3 +1,6 @@
+//go:build bdkwallet
+// +build bdkwallet
+
 package bdkgo
 
 // #include <bdkgo.h>
diff --git a/wallet/wallet.go b/wallet/wallet.go
index b6eed5a6..d677b9f3 100644
--- a/wallet/wallet.go
+++ b/wallet/wallet.go
@@ -1,3 +1,6 @@
+//go:build bdkwallet
+// +build bdkwallet
+
 package wallet
 
 //#cgo LDFLAGS: ./wallet/bdkgo_lib/target/release/libbdkgo.a -ldl -lm
diff --git a/wallet/wallet_test.go b/wallet/wallet_test.go
index 3b804122..061291b9 100644
--- a/wallet/wallet_test.go
+++ b/wallet/wallet_test.go
@@ -1,3 +1,6 @@
+//go:build bdkwallet
+// +build bdkwallet
+
 package wallet
```

The relevant files are `bdkgo.go`, `wallet.go`, and `wallet_test.go`